tests/robustness/failpoint/failpoint.go

```diff
@@ -29,10 +29,7 @@ import (
 )
 
 const (
-	triggerTimeout               = time.Minute
-	waitBetweenFailpointTriggers = time.Second
-	failpointInjectionsCount     = 1
-	failpointInjectionsRetries   = 3
```

Comment:
why is the retries removed? was it ever useful?

Reply:
It was introduced as a band aid solution to flakes. It reduced problem a little, but there were still some issues with failpoint that retries didn't help. Over time we improved, especially after it we discovered issue with process execution. 

Hard to say if removing it will expose some issues, we should see them in CI.

Reply:
Checked logs form one of robustness test runs. Didn't find any "Failed to trigger failpoint" logs.